src/routes/ExercisesScreens.tsx

```diff
@@ -17,6 +20,17 @@ const Root = styled.View`
   background-color: ${prop => prop.theme.colors.background};
   height: 100%;
 `
+const SmallMessage = styled(HeadingText)`
```

Comment:
These predefined texts - like `HeadingText` are used, so you do not need to set font-size and font-weight everywhere. So please have a look at `src/component/text` and use the one that is appropriate for your usecase, in this case you should use `ContentTextLight` and then you can remove font-size and font-weight